tools/deploy_assisted_installer.py

```diff
@@ -1,14 +1,24 @@
 import os
 import utils
+import argparse
+
+
+parser = argparse.ArgumentParser()
+parser.add_argument("--deploy-tag")
```

Comment:
Consider adding
```suggestion
parser.add_argument("--deploy-tag", help='Tag for all deployment images', type=str, default='latest')
```
Will save you some if statements in the code.

Reply:
Because i'm using it in the Makefile it will be complicated to have functions that will determinate if the flag was set or not, so in the makefile the default value is "" (empty string) in this case the default value that i set here will not be used. for example
```
python bla.py  --deploy-tag ""
Namespace(create_namespace=False, deploy_tag='', domain=None)
```
And currently everyone are using stable and not latest, except the UI. 
But i don't mind adding it if it will help you in the future.

Reply:
Done, added.